backend/workflow_manager/workflow_v2/execution.py

```diff
@@ -241,7 +265,7 @@ def publish_initial_workflow_logs(self, total_files: int) -> None:
         )
 
     def publish_final_workflow_logs(
-        self, total_files: int, processed_files: int
+        self, total_files: int, successful_files: int, failed_files: int
```

Comment:
@vishnuszipstack DOn't we have to modify the execute() function above to include params 
run_id: str, file_name: str ?

Reply:
@gaya3-zipstack It might be came after rasing this PR. We can address it another PR or along this PR. both are acceptable.